CompanionApp/CompanionApp/Models/ExamplableOrganisms.swift

```diff
@@ -444,26 +444,15 @@ extension Components.Organisms.InformationMessageCard: Examplable {
                     icon: UIImage(named: "warning")!,
                     headline: .init(
                         title: "Info message",
-                        ctas: nil
+                        body: "This is where we can have a short bit of copy about this thing.",
+                        ctas: [ .init(
+                            message: "Do something",
+                            link: "",
+                            linkType: .normal,
+                            displayType: .primary
+                        )]
                     ),
-                    bodyContent: .init(
```

Comment:
Would it be worth keeping at least one example that uses `bodyContent` for the Companion App?

Reply:
Marc and Adam said that format was message was out of date now

Reply:
Shall we just remove the `bodyContent` parameter alltogether to avoid confusion?